lua/lspconfig/util.lua

```diff
@@ -428,4 +428,20 @@ function M.strip_archive_subpath(path)
   return path
 end
 
+-- Replace with built in vim.version tostring that was added in neovim v0.10
+-- Used mainly for printing out of date version, so after v0.10 is no longer
+-- supported at least
+---@param version vim.Version
+function M.version_string(version)
```

Comment:
don't want a new "public" API, please make this a `local function` in lspconfig.lua

Reply:
This is true, I moved it to being a local function. I also added checking to make sure the minimum version is able to be parsed as well